sass/sass_wrapper.js

```diff
@@ -0,0 +1,46 @@
+const {debug, log, runAsWorker, runWorkerLoop} = require('./worker');
+const sass = require('sass');
+const fs = require('fs');
+
```

Comment:
Add a file-level comment that explains the purpose of this program?

Reply:
Done.